CHANGES.md

```diff
@@ -1,3 +1,7 @@
+## 0.0.36
```

Comment:
We're only up to 0.0.35!!

Reply:
Whoops. For some reason I thought 0.0.35 was skipped with the double release you did to prod. I think because I'm used to the update-version-number-on-release we use in TerriaJS and there are v0.0.35 commits everywhere.